airflow/providers/yandex/operators/dataproc.py

```diff
@@ -37,6 +37,25 @@ class InitializationAction:
     timeout: int  # Execution timeout
 
 
+# These two functions are needed, since when render_as_native_obj=True, Jinja can deduce wrong types.
+# So we force elements to be string in API parameters
+# see https://github.com/apache/airflow/discussions/26336
+
+
+def force_dict_values_as_strings(dct: Optional[Dict[str, Any]]) -> Optional[Dict[str, str]]:
+    if dct is None:
+        return None
+
+    return {k: str(v) for k, v in dct.items()}
+
+
+def force_list_values_as_string(lst: Optional[Iterable[Any]]) -> List[str]:
+    if lst is None:
+        return None
+
+    return list(map(str, lst))
+
+
```

Comment:
`from airflow.utils.template import literal`

https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/operators.html#jinja-templating

Reply:
Could you, please, elaborate on this?
From the documentation, 

> We can prevent airflow from treating this value as a reference to a file by wrapping it in literal(). This approach disables the rendering of both macros and files and can be applied to selected nested fields while retaining the default templating rules for the remainder of the content.

But preventing from templating is not what I want.
The problem is, when some onepass, for example, 
```
DataprocCreatePysparkJobOperator(..., args = ['{{params["parameter_with_int_value"]}}'])
```

Jinja Templating with `render_template_as_native_obj=True` will convert  `'{{params["parameter_with_int_value"]}}'` to `int` and the API call fails, since it is required that `args` is a list of strings.